README.md

```diff
@@ -46,7 +45,11 @@ VH:::mg --> DTS-VH-PROD
 
 The diagram above is generated automatically from the subscriptions listed in `environments/prod/prod.tfvars` via a [GitHub Action](.github/workdlows/create-mermaid-diagram.yml).
 
-If you add a new management group, be sure to update the [script](scripts/create-mermaid-diagram.sh#L34-L37) so the desired structure is accurately reflected in the diagram.
+If you add a new management group and subscriptions to `prod.tfvars`, this will be automatically reflected in the diagram.
+
+If you need any custom formatting, you must edit the script.
```

Comment:
could probably drop these.

1. seems leftover from when it wasn't
2. custom formatting you obviously need to edit it

```suggestion
```